src/Mooltipass/MPNodeBLE.cpp

```diff
@@ -35,7 +35,33 @@ bool MPNodeBLE::isValid() const
     }
 
     return (data.size() == PARENT_NODE_LENGTH && (NodeParent == type || NodeParentData == type)) ||
-           (data.size() == CHILD_NODE_LENGTH && (NodeChild == type || NodeChildData == type));
+            (data.size() == CHILD_NODE_LENGTH && (NodeChild == type || NodeChildData == type));
+}
+
+QByteArray MPNodeBLE::getLoginNodeData() const
+{
+    // return core data, excluding linked lists, flags and last child node used address
+    if (!isValid()) return QByteArray();
+    return data.mid(DATA_ADDR_START, LAST_CHILD_NODE_USED_ADDR_START - DATA_ADDR_START);
```

Comment:
I don't get that one at all, when it used to be 
`return data.mid(DATA_ADDR_START);`